Update virtual-machines-disks-encryption-at-host-restrictions.md (#120688)
Azure/azure-content: includes/virtual-machines-disks-encryption-at-host-restrictions.md

- Supported for 4k sector size Ultra Disks and Premium SSD v2.
- Only supported on 512e sector size Ultra Disks and Premium SSD v2 if they were created after 5/13/2023.
    - For disks created before this date, [snapshot your disk](../articles/virtual-machines/disks-incremental-snapshots.md) and create a new disk using the snapshot.

> - Can't be enabled if migrating a virtual machine or a virtual machine scale set that is encrypted with Azure Disk Encryption, or has ever been encrypted with Azure Disk Encryption, to Encryption at Host. See [Azure Disk Encryption scenarios on Windows VMs](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/virtual-machines/windows/disk-encryption-windows.md#restrictions) for further restrictions and unsupported scenarios when using Azure Disk Encryption.

Reviewer: ```suggestion
- Can't be enabled on virtual machines (VMs) or virtual machine scale sets that currently or ever had Azure Disk Encryption enabled. 
```

Author: @roygara Can we not reference another article? The reason I put that in there is that there's a whole load of other information which maybe relevant to some customers. Had this been referenced then it would have prevented one of my customers from moving away from ADE.

Reviewer: @mikewarr I reviewed that information. It's not relevant to the subject of this include or the articles it appears in. So we won't include it here. We try to keep content in articles as focused as possible. :)
